<?xml version="1.0" ?>
<robot name="reach_study" xmlns:xacro="http://ros.org/wiki/xacro">
  <xacro:include filename="$(find reach_ros)/demo/model/motoman_sia20d/motoman_sia20d_macro.xacro" />
  <xacro:motoman_sia20d prefix=""/>

  <link name="reach_object"/>
  <link name="tcp"/>

  <joint name="robot_to_reach_object" type="fixed">
    <parent link="base_link"/>
    <child link="reach_object"/>
    <origin xyz="0.75 0 0.2" rpy="0 ${radians(90)} ${radians(180)}"/>
  </joint>
  <joint name="tool0_to_tcp" type="fixed">
    <parent link="tool0"/>
    <child link="tcp"/>
    <origin xyz="0 0 0.1" rpy="0 0 0"/>
  </joint>
</robot>
